docs/index.md

```diff
@@ -54,5 +54,5 @@ could be wrong. So starting with number 1 is never a bad idea.
 
 ## Additional features
 
-* [Web proxy: Tinyproxy](web-proxy.md)
+* [Web proxy: Privoxy](web-proxy.md)
```

Comment:
Can you please also update the web-proxy.md with any necessary documentation changes? This can be done separately though

Reply:
Have done with https://github.com/jay-to-the-dee/docker-transmission-openvpn/commit/f07e865be5d9e0d47fc40f6b3ba65ddb45b9c6b3 :)